docs/request-audit-headers.md

```diff
@@ -25,28 +25,13 @@ For synchronous (i.e. non-bulk) requests, BFD users SHALL include meaningful val
 * `BlueButton-UserId`: the unique ID in the frontend system for the user (e.g. beneficiary) that data is being requested on behalf of.
 * `BlueButton-User`: the human-readable login ID (e.g. email address) in the frontend system for the user (e.g. beneficiary) that the data is being requested on behalf of.
 * `BlueButton-BeneficiaryId`: the unique beneificary ID (e.g. `Patient.id`) from BFD for the user the beneficiary whose data is being requested.
+* `x-forwarded-for`: a standard header for identifying the originating IP address of a client connecting to a web server through an HTTP proxy or a load balancer.
 
 ## Asynchronous Requests
 
 For asynchronous (i.e. bulk) requests, BFD users SHALL include meaningful values for as many as possible of the following HTTP headers.
 
-### BCDA
-
-* `BlueButton-OriginalQueryId`: (see above; same thing)
-    * For asynchronous jobs, users SHALL use the `BlueButton-OriginalQueryId` for the HTTP request that created/submitted/whatever the job.
-* `BCDA-CMSID`: the 5 digit A1234 Identifier for ACOs
-* `BCDA-JOBID`: integer generated by DB sequence
-
-### AB2D
-
-* `BlueButton-OriginalQueryId`: (see above; same thing)
-    * For asynchronous jobs, users SHALL use the `BlueButton-OriginalQueryId` for the HTTP request that created/submitted/whatever the job.
-* `AB2D-CMSID`: the Part D contract ID
-* `AB2D-JOBID`: integer generated by DB sequence
-
-### DPC
-
 * `BlueButton-OriginalQueryId`: (see above; same thing)
     * For asynchronous jobs, users SHALL use the `BlueButton-OriginalQueryId` for the HTTP request that created/submitted/whatever the job.
-* `DPC-CMSID`: The NPI of the downstream requesting provider or organization.
-* `DPC-JOBID`: integer generated by DB sequence
+* `BULK-CLIENTID`: a unique identifier of the bulk client for whom this request is for (i.e. ACO ID, NPI, Part D Contract)
```

Comment:
DPC uses UUIDs as internal identifiers, as mentioned elsewhere in another comment. DPC logging always includes the UUID, not the NPI. Any problem with using UUID here?

Reply:
This will be ok for us. Thanks.